CHANGELOG.md

```diff
@@ -5,6 +5,19 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.7.1] - 2021-12-16
+# Fixed
+- Vulnerable log4j package updated to prevent remote code execution [#55](https://github.com/IN-CORE/incore-services/issues/55)
```

Comment:
Proposed amendment:

- Vulnerability related to log4j packaged addressed to prevent remote code execution

Reply:
I'll keep this in mind to adjust in next release. I can't add this in the merge PR, that is only meant for sync-ing.

I created this PR with wrong branch. Closed this one and created another